servatrice/migrations/servatrice_0024_to_0025.sql

```diff
@@ -0,0 +1,6 @@
+-- Servatrice db migration from version 24 to version 25
+
+ALTER TABLE `cockatrice_uptime` ADD COLUMN `mods_count` int(11) NOT NULL DEFAULT 0;
+ALTER TABLE `cockatrice_uptime` ADD COLUMN `mods_list` TEXT;
+
+UPDATE `cockatrice_schema_version` SET `version` = 25 WHERE `version` = 24;
```

Comment:
The build script is checking for an exact string here; the check fails because the table and column names are surrounded by ` quotes.
Quick fix: remove them

Reply:
This has been done with other sql migrations... oh well